mscore/prefsdialog.cpp

```diff
@@ -102,6 +102,12 @@ PreferenceDialog::PreferenceDialog(QWidget* parent)
       exportAudioSampleRate->addItem(tr("44100"), 44100); // default
       exportAudioSampleRate->addItem(tr("48000"), 48000);
 
+      zoomType->clear();
+      zoomType->addItem(tr("Page Width"), 0);
+      zoomType->addItem(tr("Whole Page"), 1);
+      zoomType->addItem(tr("Two Pages"), 2);
+      zoomType->addItem(tr("Percentage"), 3);
```

Comment:
Where does this stem from? That should ne in already, from @worldwideweary's recent changes?

Reply:
Unfortunately it didn't work. So I checked how the rest of the QComboBox widgets were setup and this is the way.

Reply:
Then those strings might get removed (or marked `notr="true"`) elsewehere?

Reply:
Done, removed them to not confuse future preference additions.

Reply:
For better look in QtDesigner, I'd rather leave those strings in but mark notr="true", maybe with a remark ' translated elsewhere'